valhalla/jawn/src/managers/score/ScoreManager.ts

```diff
@@ -60,4 +67,14 @@ export class ScoreManager extends BaseManager {
       return err(error.message);
     }
   }
+
+  private mapScores(scores: Scores): Score[] {
+    return Object.entries(scores).map(([key, value]) => {
+      return {
+        score_attribute_key: key,
+        score_attribute_type: typeof value === "boolean" ? "boolean" : "number",
```

Comment:
can a score also be a "string"?

Reply:
for now just bool and number for QAWolf case